src/Microsoft.VisualStudio.Services.Agent/HostContext.cs

```diff
@@ -766,7 +784,7 @@ public static void AddAdditionalMaskingRegexes(this IHostContext context)
             ArgUtil.NotNull(context, nameof(context));
             foreach (var pattern in AdditionalMaskingRegexes.CredScanPatterns)
             {
-                context.SecretMasker.AddRegex(pattern, $"HostContext_{WellKnownSecretAliases.CredScanPatterns}");
+                context.SecretMasker.AddRegex(pattern, $"HostContext");
```

Comment:
Revert. There was some discussion offline a while ago that this would print the collection type name, but WellKnowSecret*Aliases*.* are strings.